lua/sql/table.lua

```diff
@@ -0,0 +1,204 @@
+local t = {}
+t.__index = t
+
+function t:__run(func)
+  if self.db.closed then
+    return self.db:with_open(function()
+      return func() -- shoud pass tbl name?
+    end)
+  else
+   return func()
+  end
+end
+
+function t:__clear_cache(succ, change)
+  if succ then self.cache = {} end
+end
+
+function t:__fill_cache(query, res)
+ return table.insert(self.cache, {query, res}) -- should be kv instead?
+end
+
+function t:__get_from_cache(query)
```

Comment:
Why are you not working :(. 

I'm going to implement memorize here so if the query is requested from cache once, the request will be cached